lxd/resource_lxd_container.go

```diff
@@ -328,17 +329,6 @@ func resourceLxdContainerCreate(d *schema.ResourceData, meta interface{}) error
 	// Container has been created, store ID
 	d.SetId(name)
 
-	d.SetPartial("name")
-	d.SetPartial("image")
-	d.SetPartial("profiles")
-	d.SetPartial("project")
-	d.SetPartial("ephemeral")
-	d.SetPartial("privileged")
-	d.SetPartial("config")
-	d.SetPartial("limits")
-	d.SetPartial("device")
-	d.SetPartial("remote")
```

Comment:
Does `SetPartial` no longer work with the SDK update? If it still works, let's keep this and remove it in another PR (in trying to keep PRs scoped to a specific focus).